includes/Modules/Ads.php

```diff
@@ -126,8 +162,19 @@ protected function setup_assets() {
 					},
 				)
 			);
-
-			if ( current_user_can( Permissions::VIEW_AUTHENTICATED_DASHBOARD ) && $this->is_connected() ) {
+			// Integrator should be included if either Ads module is connected already,
+			// or we are on the Ads module setup screen.
+			if (
+				current_user_can( Permissions::VIEW_AUTHENTICATED_DASHBOARD ) &&
+				(
+					// Integrator should be included if either:
+					//
+					// The Ads module is already connected.
+					$this->is_connected() ||
```

Comment:
This is not entirely accurate as a check since the module isn't guaranteed to be active at this point (assets are set up for all available modules, regardless of active state). It should be sufficient in practice since the module's settings should be removed when deactivated but this kind of thing has been a problem before. TL;DR, it's probably fine in this instance, but worth being aware of.